packages/pn-pa-webapp/src/pages/ApiKeys.page.tsx

```diff
@@ -139,6 +167,16 @@ const ApiKeys = () => {
     void dispatch(deleteApiKey(apiKeyId)).then(() => void dispatch(getApiKeys()));
   };
 
+  // Pagination handlers
+  const handleChangePage = (paginationData: PaginationData) => {
+    trackEventByType(TrackEventType.NOTIFICATION_TABLE_PAGINATION);
```

Comment:
this track event is wrong. this function logs on a platform called mixpanel and with this event we are logging that an event of type "notification pagination" occurred

Reply:
Yeah i actually forgot to replace them even though i actullay added those 2 new type of trackEvent.

Reply:
changes made into 5eb41f24bacd7862f6c9ac99ae9f9e9396e71a40